housecost.js

```diff
@@ -0,0 +1,25 @@
+let volumeInMeters=8*10*10;
+let gardenSizeInM2=100;
+let peterHouseCost=2500000;
+
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+console.log(housePrice);
+if (housePrice < peterHouseCost) {
+    console.log("peter is paying too much amount"); 
+} 
+else {
+    console.log("peter is paying too little amount");
+}
+
+
+volumeInMeters=5*11*8;
+gardenSizeInM2=70;
+juliaHouseCost= 1000000;
```

Comment:
you don't have any variable called `juliaHouseCost` you should declare it before even if it works but still better.
```js
const juliaHouseCost = 1000000;
```